app/views/verify/_hardfail4.html.slim

```diff
@@ -1 +1,11 @@
-p.mb1 = t('idv.messages.hardfail4', sp: decorated_session.sp_name)
+p.mb1 = t('idv.messages.hardfail', hours: Figaro.env.idv_attempt_window_in_hours)
+.mt2.p3.col-12.center.border-box.border.border-teal.rounded-xl
+  p.mb3.fs-20p = t('idv.messages.hardfail4_html', sp: decorated_session.sp_name)
+  = link_to t('idv.buttons.continue_plain'),
+    decorated_session.sp_return_url,
```

Comment:
I'm not 100% convinced this is the right URL
- for SAML, this URL will be the "The URL of the SP which login.gov provides to users when they wish to go directly to the SP site or cancel out of authentication."
- for OIDC this will be the decline/cancel URL which will probably show an error

I believe our goal is to have SP register a *new* URL specifically as a help page, so we'll need to add a new migration, update fields in the dashboard, etc etc

I am OK with us merging this as-is but it's wrong and we'll need to plan to fix it in the short-term

Reply:
@nickbristow let's talk about how to fix this when you get back

Reply:
@zachmargolis you are correct, we don't have the SP's registering a help url atm, I think thats planned.